packages/components/src/snackbar/style.scss

```diff
@@ -28,18 +28,26 @@
 }
 
 .components-snackbar__action.components-button {
-	margin-left: 32px;
+	margin-left: $grid-size * 4;
 	color: $white;
 	height: auto;
 	flex-shrink: 0;
 	line-height: $default-line-height;
+	padding: 0;
 
 	&:not(:disabled):not([aria-disabled="true"]):not(.is-default) {
 		text-decoration: underline;
+		background-color: transparent;
 
-		&:hover {
+		&:focus {
 			color: $white;
-			text-decoration: none;
+			box-shadow: none;
+			outline: 1px dotted $white;
+		}
+		
+		&:hover,
```

Comment:
@jasmussen, this applies the blue color also when it's both hovered and focused:

![Screen Shot 2019-10-14 at 14 05 58](https://user-images.githubusercontent.com/699132/66750745-8921bd80-ee8d-11e9-9d7d-16dd77ea7b0c.png)

Reply:
This looks good.

Instead of this:

```
		&:focus {
			color: $white;
			box-shadow: none;
			outline: 1px dotted $white;
		}
		
		&:hover {
			color: $blue-medium-400;
		}
```

Is this sufficient?

The idea being that because the `hover` rule is registered _after_ the focus rule, it overrides it without needing extra specificity?

If that doesn't work, then this is fine!

Reply:
it didn't work for some reasons 🤷‍♂ 

the very first thing I tried

Reply:
Well, I tested again and it works 🙂 Will update PR.